frontend/packages/ceph-storage-plugin/src/components/modals/volume-snapshot-modal/volume-snapshot-modal.tsx

```diff
@@ -29,17 +36,130 @@ export type VolumeSnapshotModalProps = {
 export const VolumeSnapshotModal = withHandlePromise((props: VolumeSnapshotModalProps) => {
   const { close, cancel, pvcData, errorMessage, inProgress, handlePromise } = props;
   const resource = pvcData.data as K8sResourceKind;
+  const [snapshotCount, setSnapshotCount] = React.useState(3);
```

Comment:
Why is the initial value 3? Maybe a comment or a constant name that would enlighten us about the choice,

Reply:
And better to create a constant for the same. May be in same file or in constants folder